tests/test_cli.py

```diff
@@ -55,6 +56,127 @@ def raises_exception():
         raises_exception()
         mock_exit.assert_called_once_with(1)
 
+    def test_single_var(self):
```

Comment:
These test could probably be parametrized (@pytest.mark.parametrize) - there is an example on line 172.

Reply:
Okay I gave it a go. Although the parametrized solution is less LOC, i'm not sure it's easier to read.